app/forms/requested_document_upload_form.rb

```diff
@@ -9,10 +9,11 @@ def initialize(documents_request, *args, **kwargs)
 
   def save
     document_file_upload = attributes_for(:documents_request)[:document]
+    document_type = attributes_for(:documents_request)[:document_type]
     if document_file_upload.present?
       @documents_request.documents.create(
         uploaded_by: @documents_request.intake.client,
-        document_type: DocumentTypes::RequestedLater.key,
+        document_type: document_type || DocumentTypes::RequestedLater.key,
```

Comment:
so was it just not using `UploadDocumentsController#document_type` before? would love to go over this